instrumentation/action_pack/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History: opentelemetry-instrumentation-action_pack
 
+### v0.2.1 / 2022-10-04
+
+* (No significant changes)
```

Comment:
* FIXED: Ensures the correct route is add to http.route span attribute 
* FIXED: bring http.request.header and http.response.header in line with semantic conventions